src/angular/3-creating-components/creating-components.md

```diff
@@ -332,28 +307,43 @@ For now, we'll use fake data for a list of restaurants in the component, and put
 
 @highlight 3-97, 104, 109-111
 
+To solve this problem, use Angular directives to iterate through data and display properties of restaurants.
+
 ✏️ Update **src/app/restaurant/restaurant.component.html** to be:
 
 ```html
 <div class="restaurants">
   <h2 class="page-header">Restaurants</h2>
   <!-- if restaurants has a length show the list -->
 
-  <!-- inside this container, show the following markup for each restaurant -->
+  <!-- show the following markup for each restaurant -->
   <div class="restaurant">
-
+    <img src="{{ restaurant.images.thumbnail }}" width="100" height="100" />
```

Comment:
At some point, we need to add `alt` attributes to our images.

Reply:
absolutely, a full review of the code with an accessibility concern would be nice when we have time